module_auto_update/wizards/module_upgrade.py

```diff
@@ -8,6 +8,16 @@
 class ModuleUpgrade(models.TransientModel):
     _inherit = 'base.module.upgrade'
 
+    @api.model
+    def get_module_list(self):
```

Comment:
Ahhhh so that's what caused this - I noticed it the other day then just kind of shrugged it off.